util/pkg/distributions/identify_test.go

```diff
@@ -99,6 +99,11 @@ func TestFindDistribution(t *testing.T) {
 			err:      nil,
 			expected: DistributionUbuntu2004,
 		},
+		{
+			rootfs:   "ubuntu2010",
+			err:      nil,
+			expected: DistributionUbuntu2010,
+		},
```

Comment:
The tests expect the `/etc/os-release` from an Ubuntu 20.10 to be copied to the `tests` dir.